docs/fastdds/rpc_dds/rpc_dds_intro.rst

```diff
@@ -6,18 +9,43 @@ RPC over DDS
 .. _RPC over DDS specification: https://www.omg.org/spec/DDS-RPC/1.0/PDF
 
 *Remote Procedure Calls* (see `RPC over DDS specification`_), also known as RPC,
-is a type of bidirectional communication used in a request-reply pattern.
+is a type of bidirectional communication, used in a request-reply pattern, that can be implemented
+using DDS entities such as DomainParticipants, Topics, DataWriters and DataReaders.
+
 The RPC architecture is based on the client-server model: the client sends a request to the server,
-and the server sends one or more responses (replies) back to the client.
-In the context of Data Distribution Service (DDS), the client and server are modeled using DomainParticipants
-and they contain all the necessary objects to communicate with each other.
-Each client (internally represented by a *Requester* entity) sends a request sample
-to the server through a request topic;
-the server (internally represented by a *Replier* entity) processes it and sends a reply sample back to the client
-through a different topic (reply topic).
-All the Requesters and Repliers internally have a DataWriter and DataReader to send and receive samples.
-
-.. toctree::
```

Comment:
I'd prefer the way it was before. Just the toctree here and then the a separate pages for the low level API, the exceptions and another for the streaming.